content/engineering/bruteforce-lock-and-unlock/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Testing Brute force lockout using LoginRadius"
+date: "2024-05-08"
+coverImage: ""
+author: "Gayathri Suresh"
+tags: ["Engineering","API","Identity Experience Framework", "Login", "Authentication", "Brute force lockout"]
+description: "In this blog, we'll learn about brute force lockout, creation of basic app using Identity Experience Framework, and on how to unlock an user account using APIs."
+---
+
+
+# Brute force lockout and unlock
+
+---
+
+## Contents
+- [Basic Terminologies](#basic-terminologies)
+- [API Implementation](#api-implementation)
+    - [Creating a Basic Application](#creating-a-basic-application)
+    - [Testing Brute force lockout](#testing-brute-force-lockout)
+    - [Lockout types supported by LoginRadius](#lockout-types-supported-by-loginradius)
+    - [Unlocking an account locked through brute force lockout](#unlocking-an-account-locked-through-brute-force-lockout)
+        - [Account Update API from the LoginRadius Account API collections](#account-update-api-from-the-loginradius-account-api-collections)
+        - [Auth Unlock Account by Access Token from the LoginRadius Authentication API collections](#auth-unlock-account-by-access-token-from-the-loginradius-authentication-api-collections)
+- [Conclusion](#conclusion)
+- [Miscellaneous](#miscellaneous)
+    - [CAPTCHA](#captcha)
+    - [MFA](#mfa)
+
+---
+
+## Basic Terminologies
+1. **Brute force attack:**
+- A method where every possible combination of characters or values is systematically tried to gain unauthorized access to a system, application, or data.
+
+2. **Brute force lock:**
+- Brute force lock is a type of account lock made to prevent a bruteforce attack.
+
+3. **Brute force lockout:**
+- Brute force lockout is a security mechanism that blocks access after a certain number of failed authentication attempts to prevent unauthorized access through repeated trial and error.
+
+4. **CAPTCHA (Completely Automated Public Turing test to tell Computers and Humans Apart):**
+- CAPTCHA is a method used to determine whether a user is human or not by presenting a challenge that is easy for humans to solve but difficult for bots.
+
+5. **Multi-Factor Authentication (MFA):**
+-  MFA is a security method that requires multiple forms of identification to grant access, typically combining something the user knows, has, and is.
+
+In _LoginRadius_, the brute force lockout can be implemented in two ways, using:
+1. APIs
+2. Workflows
```

Comment:
@gayathrisuresh150501 Workflows part is not discussed here, so I think we can remove this.
cc: @ravitejag